yew/src/virtual_dom/vcomp.rs

```diff
@@ -16,6 +16,13 @@ cfg_if! {
     }
 }
 
+cfg_if! {
```

Comment:
A very minor nit:
You don't *really* need the `cfg_if!` here.
```
cfg_if! {
    if #[cfg(feature = "ssr")] {
        use super::{Html, HtmlStringifyError};
        use std::convert::TryFrom;
    }
}
```
Can be written as (I think)
```
#[cfg(feature = "ssr")] {
        use super::{Html, HtmlStringifyError};
        use std::convert::TryFrom;
}
```

Reply:
I don't think this will work. The cfg attribute treats the block as an expression (i.e. the braces are part of it) so if the feature is present the compiler will receive the following:
```rust
{
  use super::{Html, HtmlStringifyError};
  use std::convert::TryFrom;
}
```
which won't compile.

It works with `cfg_if!` because the braces are part of the macro syntax and don't show up in the output.